tests/test_analyzer.py

```diff
@@ -71,3 +81,16 @@ def test_analyzer_management(db, bad_db, cluster, enterprise, db_version):
             "format": "latLngDouble",
         }
         assert db.delete_analyzer(analyzer_name)
+
+    # Test create delimieter analyzer with multiple delimiters
+    # NOTE: This is currently failing on 3.12.0-NIGHTLY.20240116
+    if db_version >= version.parse("3.12.0"):
+        result = db.create_analyzer(
+            name=generate_analyzer_name(),
+            analyzer_type="delimiter",
```

Comment:
The analyzer type here is wrong. It should be `multi_delimiter`.
Here is the reference: https://docs.arangodb.com/3.12/index-and-search/analyzers/#multi_delimiter

Reply:
thanks fixed in b48e105